homeassistant/components/voip/voip.py

```diff
@@ -357,9 +390,8 @@ def on_chunk(self, audio_bytes: bytes) -> None:
 
         if self._audio_bytes is None:
             # 16Khz, 16-bit mono audio message
-            self._audio_bytes = (
-                Path(__file__).parent / "not_configured.pcm"
-            ).read_bytes()
+            file_path = Path(__file__).parent / self.file_name
+            self._audio_bytes = file_path.read_bytes()
```

Comment:
This is doing I/O in the event loop?

Reply:
No, it's being called from `hass.async_add_executor_job`